sdk/core/Azure.Core/src/NullableResponseOfT.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace Azure
+{
+    /// <summary>
+    /// Represents a result of Azure operation.
+    /// </summary>
+    /// <typeparam name="T">The type of returned value.</typeparam>
+#pragma warning disable SA1649 // File name should match first type name
+    public abstract class NullableResponse<T>
+#pragma warning restore SA1649 // File name should match first type name
+    {
+        private const string NoValue = "The Response has no value";
```

Comment:
Why is this preferable to just leaving it as empty string?

Reply:
I think it is more explicit - similar to "this page intentionally blank".

Reply:
Another convention we have in the code base is to return `"<null>"` ... if this is sanctioned, it might read nicely in the NullableResponse type

Reply:
Yeah I would stick with either empty string or "<null>". If users need to know if the value is really null, they have to check HasValue anyway so it doesn't help much to try to make the string representation convey that.

Reply:
For that matter, should we include HasValue in the ToString here, and add back the override of ToString into ResponseOfT for back compat?

Reply:
I think the value of HasValue is implied by this message. What's the back-compat scenario you had in mind? To restore the throwing behavior, we had before?

Reply:
No, I meant that if we include HasValue in the ToString for this type, we probably would want to NOT include it for ResponseOfT to retain the existing format.

Reply:
But the format is effectively un-changed. `Response<T>` must always have a value.

Reply:
Yes, agreed. I'm saying if we were to add HasValue into the ToString of NullableResponseOfT, we would not want to include it in the ToString of ResponseOfT.